react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,22 @@
+import Topics from './Topics';
+import { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [clicked, setClicked] = useState('');
+
+  function handleTopicsOnClick(topic) {
```

Comment:
You normally would name the function without "On" in it. Think of it this way:
"TopicClick" is the event
"onTopicClick" is the property used for handling the TopicClick event
"handleTopicClick" is the handler

(Note also that you only click on one topic, so "Topic" should be singular.)